src/igvfd/types/file_set.py

```diff
@@ -1423,7 +1417,7 @@ class AuxiliarySet(FileSet):
         'notSubmittable': True
     })
     def measurement_sets(self, request, measurement_sets):
-        return paths_filtered_by_status(request, measurement_sets)
+        return paths_filtered_by_status(request, measurement_sets) or None
```

Comment:
Could be useful to keep as returning an empty array as all auxiliary sets should eventually have a measurement set and displaying an empty array could be informative.